ui/lib/core/addon/components/secret-list-header.hbs

```diff
@@ -1,14 +1,23 @@
-{{#let (options-for-backend @model.engineType) as |options|}}
+{{#let (options-for-backend @model.engineType "" @isEngine) as |options|}}
```

Comment:
adding the params for the options-for-backend, which are `export function optionsForBackend([backend, tab, isEngine]) {`. the isEngine, is new.